cmd/store/export_test.go

```diff
@@ -0,0 +1,268 @@
+				Expectation: true,
+			},
+			{
+				TupleKey:    openfga.AssertionTupleKey{User: "user:user-2", Relation: "admin", Object: "github-repo:demo"},
+				Expectation: true,
+			},
+			{
+				TupleKey:    openfga.AssertionTupleKey{User: "user:user-1", Relation: "admin", Object: "github-repo:demo"},
+				Expectation: false,
+			},
+		},
+	}
+
+	readAssertionsOptions := client.ClientReadAssertionsOptions{
+		AuthorizationModelId: openfga.PtrString("01GXSA8YR785C4FYS3C0RTG7B1"),
+	}
+
+	mockAssertionsRequest := mock_client.NewMockSdkClientReadAssertionsRequestInterface(mockCtrl)
+	mockAssertionsRequest.EXPECT().Options(readAssertionsOptions).Return(mockAssertionsRequest)
+	mockAssertionsRequest.EXPECT().Execute().Return(&assertionsResponse, nil)
+	mockFgaClient.EXPECT().ReadAssertions(context.Background()).Return(mockAssertionsRequest)
+
+	// Execute
+	output, err := buildStoreData(clientConfig, mockFgaClient)
+
+	// Expect
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedResponse := storetest.StoreData{
+		Name: "Test store",
+		Model: `model
+  schema 1.1
+
+type user
+
+type github-repo
+  relations
+    define admin: [user]
+    define viewer: [user]
+
+`,
+		Tuples: []openfga.TupleKey{
+			{
+				User:     "user:user-1",
+				Relation: "viewer",
+				Object:   "github-repo:demo",
+			},
+			{
+				User:     "user:user-2",
+				Relation: "viewer",
+				Object:   "github-repo:demo",
+			},
+			{
+				User:     "user:user-2",
+				Relation: "admin",
+				Object:   "github-repo:demo",
+			},
+		},
+		Tests: []storetest.ModelTest{
+			{
+				Name: "Tests",
+				Check: []storetest.ModelTestCheck{
+					{
+						User:   "user:user-1",
+						Object: "github-repo:demo",
+						Assertions: map[string]bool{
+							"admin":  false,
+							"viewer": true,
+						},
+					},
+					{
+						User:   "user:user-2",
+						Object: "github-repo:demo",
+						Assertions: map[string]bool{
+							"admin":  true,
+							"viewer": true,
+						},
+					},
+				},
+			},
+		},
+	}
+
+	if output.Name != expectedResponse.Name {
+		t.Errorf("Expected name %s got %s", expectedResponse.Name, output.Name)
+	}
+
+	if strings.TrimSpace(output.Model) != strings.TrimSpace(expectedResponse.Model) {
+		t.Errorf("Expected model %s\n\ngot\n\n%s", expectedResponse.Model, output.Model)
+	}
+
+	if !reflect.DeepEqual(output.Tuples, expectedResponse.Tuples) {
+		t.Errorf("Expected tuples %v\n\ngot\n\n%v", expectedResponse.Tuples, output.Tuples)
+	}
+
+	if len(output.Tests) != 1 {
+		t.Errorf("Expected 1 output test, got %d", len(output.Tests))
+	}
+
+	for _, tst := range expectedResponse.Tests {
+		for _, expectedCheck := range tst.Check {
+			found := false
+			for _, outputCheck := range output.Tests[0].Check {
+				if reflect.DeepEqual(expectedCheck, outputCheck) {
+					found = true
+					break
+				}
+			}
+
+			if !found {
+				t.Errorf("Expected check %v not found in output", expectedCheck)
+			}
+		}
+	}
```

Comment:
I couldn't just use `DeepEqual` for this as the order of the items in the arrays coming back was not guaranteed and would sometimes be different. Maybe there's a more efficient way to write this though using maps.